### PR TITLE
Add Cronos Trading Banner

### PR DESCRIPTION
**Pull request by Linhhnk**

### CODE REVIEW

File: src/components/Banner/index.tsx [hunk 1]
```diff
@@ -117,6 +121,21 @@ function Banner({
   const theme = useTheme()
 
   const ALL_BANNERS = [
+    {
+      // KyberSwap Trading Campaign Cronos
+      id: 'kyberSwap-trading-campaign-cronos',
+      start: new Date(1658793600000), // July 26, 2022 0:00:00
```
Comment (hungdoansy) [inline src/components/Banner/index.tsx:127]: You can write like this `new Date('Jul 26 2022 00:00:00 GMT+0700')`

Comment (namgold) [reply src/components/Banner/index.tsx:127]: Or `new Date(2022, 6, 26)`

File: src/components/Banner/index.tsx [hunk 2]
```diff
@@ -117,6 +121,21 @@ function Banner({
...
+      end: new Date(1659484800000), // August 3, 2022 23:59:59
+      img: isInModal
+        ? KyberSwapTradingCampaignCronosMobile
+        : w > 768
+        ? KyberSwapTradingCampaignCronosDesktop
+        : w > 500
+        ? KyberSwapTradingCampaignCronosTablet
+        : KyberSwapTradingCampaignCronosMobile,
+      link:
+        'https://kyberswap.com/campaigns?selectedCampaignId=1',
```
Comment (namgold) [inline src/components/Banner/index.tsx:137]: fix dis

File: src/components/Banner/index.tsx (hunk 1 above)
Comment (nguyenhuudungz) [reply src/components/Banner/index.tsx:127]: @hungdoansy cool

Comment (nguyenhuudungz) [reply src/components/Banner/index.tsx:127]: this's better `new Date('2022-07-22T00:00:00.000Z')`, use for gmt

File: src/components/Banner/index.tsx (hunk 2 above)
Comment (Linhhnk) [reply src/components/Banner/index.tsx]: done bro

File: src/components/Banner/index.tsx (hunk 1 above)
Comment (Linhhnk) [reply src/components/Banner/index.tsx:104]: use epoch for easier read for now

Comment (nguyenhoaidanh) [reply src/components/Banner/index.tsx:104]: +1 for  @namgold

File: src/components/Banner/index.tsx (hunk 2 above)
Comment (namgold) [reply src/components/Banner/index.tsx]: image
Click `Resolve conversation` when u done

File: src/components/Banner/index.tsx (hunk 1 above)
Comment (Linhhnk) [reply src/components/Banner/index.tsx:104]: Note for next time :tien-sword:

Comment (nguyenhuudungz) [reply src/components/Banner/index.tsx:104]: when next time...

Comment (namgold) [reply src/components/Banner/index.tsx:104]: or `Date.UTC(year, month, day, hour, minute, second, millisecond)`

**Review by namgold — APPROVED** [review]

**Pull request merged**